src/completion/list_ddclient_protocols.sh

```
@@ -14,4 +14,4 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-echo -n $(ddclient -list-protocols | grep  -vE 'cloudns|porkbun')
+echo -n $(ddclient --list-protocols | grep  -vE 'cloudns|directnic|emailonly')
```

Shellcheck (0.11.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)